shell/platform/embedder/embedder.cc

```diff
@@ -2201,6 +2261,10 @@ FlutterEngineResult FlutterEngineRemoveView(FLUTTER_API_SYMBOL(FlutterEngine)
         "Remove view info was invalid. The implicit view cannot be removed.");
   }
 
+  // TODO(loicsharma): Return an error if the engine was initialized with
+  // callbacks that are incompatible with multiple views.
+  // https://github.com/flutter/flutter/issues/144806
```

Comment:
It might be useful to give a bit more concrete detail on what these might be in the linked issue -- right now it's just a checkbox that says roughly the same thing as here.

Reply:
The list of these callbacks is hidden in the multi-view embedder API design doc. I added this list to the tracking issue to make this discoverable :)